packages/web3-providers-ws/src/helpers.js

```diff
@@ -1,8 +1,9 @@
 var isNode = Object.prototype.toString.call(typeof process !== 'undefined' ? process : 0) === '[object process]';
+var isRN = typeof navigator !== 'undefined' && navigator.product === 'ReactNative';
```

Comment:
Looks like [the correct way](https://stackoverflow.com/questions/39468022/how-do-i-know-if-my-code-is-running-as-react-native) to check if running ReactNative